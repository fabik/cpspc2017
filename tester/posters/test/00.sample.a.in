8 11 3 4 13
1 1 5 5
7 7 5 4
4 6 2 7
1 1 7 8 0
2 2 4 3 0
3 4 6 7 0
2 9 3 10 0
