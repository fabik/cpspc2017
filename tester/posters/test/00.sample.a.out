24
2
6
0
